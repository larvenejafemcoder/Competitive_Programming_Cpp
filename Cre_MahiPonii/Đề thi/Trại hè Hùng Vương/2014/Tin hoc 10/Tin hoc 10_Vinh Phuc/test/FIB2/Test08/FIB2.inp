10
6 4
7 14
4 3
10 76
10 16
3 3
2 1
6 6
5 2
4 5
