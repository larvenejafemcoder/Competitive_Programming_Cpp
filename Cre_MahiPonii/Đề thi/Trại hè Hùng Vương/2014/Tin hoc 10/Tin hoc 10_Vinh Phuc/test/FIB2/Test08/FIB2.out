2
5
1
29
6
1
1
2
1
2
